templates/parts/mini-cart.html

```diff
@@ -1,5 +1,5 @@
-<!-- wp:woocommerce/mini-cart-contents -->
-<div class="wp-block-woocommerce-mini-cart-contents">
+<!-- wp:woocommerce/mini-cart-contents {"align":"center","className":"wc-block-mini-cart__contents"} -->
```

Comment:
Can you explain why do we need `align` here? The Mini Cart Contents block doesn't support alignment AFAIK.

Reply:
I forgot to remove it :/ 

I fixed it with [2d6bd5d](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5800/commits/2d6bd5df9132bfd56b489b108706bed89def6f9a)